web/src/main.js

```diff
@@ -253,7 +244,6 @@ function updateCo2Scale() {
   if (typeof zoneMap !== 'undefined') zoneMap.setCo2color(co2ColorScale, theme);
 }
 
-d3.select('#checkbox-colorblind').node().checked = getState().application.colorBlindModeEnabled;
```

Comment:
are you sure this can be removed?

Reply:
Ah yes, I must have removed it by mistake :man_facepalming: 

Reverted in https://github.com/tmrowco/electricitymap-contrib/pull/2317/commits/e5d0ac88d8b48e76817b96219251624b2cca2497, good catch! :+1:

Reply:
Actually it seems that there was a reason why I originally removed that line - other changes introduced in this PR make some parts of the DOM render conditionally so when initially landing on a page that doesn't contain this checkbox, we'd hit an error trying to set its state from here.

I just made a fix in https://github.com/tmrowco/electricitymap-contrib/pull/2317/commits/1ac5fa36585255dc7b31c405647ef441090990a3 by adding `checked` and `onChange` triggers directly to the checkbox element in React.

Reply:
much better!